### DISCUSSION TITLE
Generate readable snapshots (Markdown)

### DISCUSSION THREAD

**Discussion by egoist** [vitest-dev/vitest#558] [Ideas]
This is a feature from AVA, it's much more readable than a `.snap` file:

<img width="1139" alt="CleanShot 2022-01-17 at 22 02 46@2x" src="https://user-images.githubusercontent.com/8784712/149782659-04706d26-3f25-45d6-9ecd-3aa23d182629.png">

**Comment by labcorg** [comment]
Regarding snap diffs readability this rust [package](https://github.com/Wilfred/difftastic) seems interesting, enable diffs based on syntax which would be pretty neat for vitest.
